test/adapter_test.py

```diff
@@ -11,7 +8,6 @@
 from iota import BadApiResponse, InvalidUri, TryteString
 from iota.adapter import API_VERSION, HttpAdapter, MockAdapter, \
   resolve_adapter, async_return
-from six import BytesIO, text_type
```

Comment:
❔ I think there's also a type hint (`BytesIO`) that needs to be updated somewhere in this module.

Reply:
I can't find it, maybe unused import?

Reply:
Even better 😸